src/content/docs/explanations/language/statements.mdx

```diff
@@ -7,6 +7,34 @@ functions on data streams. Each operator can be thought of as a modular unit
 that processes data and can be combined with other operators to create complex
 workflows.
 
+[Statements](/explanations/language/statements) provide control and structure with
+bindings, operators, assignments, and control-flow primitives.
+
+## `let`
```

Comment:
I feel like we should move this further down the page and start with operators or assignments.